### PR TITLE
[issue #879] moved all installations to /opt/jderobot

### LINKED ISSUE #879: [installation] move all install to /opt/jderobot

### PR DESCRIPTION
**Pull request by aitormf**
Moved all installations to /opt/jderobot

now we need use following source commad instead of gazebo-setup:
<pre>
source /opt/jderobot/soruce.bash
</pre>

Is needed try all components, in particular VisualStates (@reysam93) and  ardrone_driver because These components have hardcoded paths and I had to modify the code.

visualStates installation is now done using cmakelists instead of setup.sh script

@fqez and @chanfr, can you verify this PR?

This PR fix #879 and #872

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -157,6 +165,9 @@ INSTALL(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/Deps DESTINATION ${CMAKE_INSTALL_P
 # Install ROS Launch files
 INSTALL(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/scripts/launch DESTINATION ${CMAKE_INSTALL_PREFIX}/share/jderobot)
 
+# Install setup.bash
+INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.bash DESTINATION ${CMAKE_INSTALL_PREFIX}/share/jderobot)
```
Comment (fqez) [inline CMakeLists.txt]: This one should be just ${CMAKE_INSTALL_PREFIX}

**Comment by aitormf** [comment]
@reysam93 updated this line, thanks

**Comment by lr-morales** [comment]
@aitormf Please check your PR branch for a possible solution for the lock problem.

**Comment by aitormf** [comment]
Thanks @lr-morales for the solution

**Pull request merged**